### PR TITLE
Updates to contribution guidelines, fixes

### PR DESCRIPTION
**Pull request by OrangeChannel**
Tested working on browser. Haven't tested on mobile but doubt anything is different.

Also tried playing around with the new gitbook format and well... the "GitHub integration" sure is [something](https://github.com/OrangeChannel/fansubbing-guide/commit/e5205b8de2340cc4a5da87dbea195e9e69525ba3). Doesn't allow for editing from non-organization members (2 people max for free) and it replaces all the markdown syntax with garbage. Plug-ins are mostly gone, there's no way to use footnotes, and glossary is gone. A lot of formatting is also gone, and theme-ing is pay-to-win.

### CODE REVIEW

**Comment by FichteFoll** [comment]
Yep, the new gitbook format is a no-go. If we ever run into something we can't solve with gitbook but would like to have regardless, we'd need to look into an alternative.

I'll review this later.

**Comment by OrangeChannel** [comment]
CSS changes in a07dd1b :

before
Before

after
After

**Comment by OrangeChannel** [comment]
Need @stuxcrystal to confirm these two new pages are fine, as they use his information, and are mostly copied from yuuno's site.

**Comment by stuxcrystal** [comment]
Perfect. Just make sure this page is accessible by at most two clicks from any page on the site.

Which, I think, is given here.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,14 +53,10 @@ why it only applies to a specific language.
...
 
 This guide is written in [Markdown][]
-and uses [gitbook][]'s [toolchain][] to compile
+and uses legacy gitbook's [toolchain][] to compile
 the static HTML pages.
 Gitbook uses the [Github Flavoured Markdown][GFM] (GFM) variant.
 
-Documentation for gitbook's toolchain (including gitbook markdown sytax)
-can be found here:
-<https://toolchain.gitbook.com/>
```
Comment (FichteFoll) [inline CONTRIBUTING.md:62]: Ah, they nuked it. Too bad.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -294,9 +330,35 @@ For a full document citation,
...
+Examples of this can be found on the
+[Aegisub and Other Tools](typesetting/aegisub.md) page.
+
+
+### Footnotes
+
+Footnotes can be used for information that would interrupt
+the flow or purpose of a paragraph,
+but may still be of interest.
+They are created with `[^#]` in-text,
```
Comment (FichteFoll) [inline CONTRIBUTING.md:350]: An explanation that `#` should be replaced by a number is necessary here.

File: typesetting/aegisub.md [hunk 3]
```diff
@@ -28,12 +28,10 @@ Here is a list of tools you will want to download:
           - [MainType][]
           - [Suitcase Fusion][]
   - Software for motion-tracking
-      - Option 1: [Mocha Pro standalone app][]
+      - [Mocha Pro standalone app][]
         - Look for it on *certain websites*.
         - On Windows, Mocha **requires** [Quicktime][] to be installed.
           More information can be found [here][quicktimeFAQ].
-      - Option 2: [Blender][][^1]
```
Comment (FichteFoll) [inline typesetting/aegisub.md:35]: While Blender is a valid option, I hardly know anyone who knows how to use it, let along be willing to add it to this guide. I guess less is more in this case?

File: overview/roles.md [hunk 4]
```diff
@@ -90,20 +90,20 @@ presentation of translated text on-screen. These are generally called *signs*.
 
 For example, given this scene and a translation of "Adachi Fourth Public High School"…
 
-![\[DameDesuYo\] Eromanga-sensei - 01 (1920x1080 10bit AAC) \[05CB518E\].mkv\_snapshot\_03.11\_\[2017.08.18\_21.14.55\].jpg](images/cnvimage100.png)
+![[DameDesuYo] Eromanga-sensei - 01 (1920x1080 10bit AAC) [05CB518E].mkv_snapshot_03.11_[2017.08.18_21.14.55].jpg](images/cnvimage100.png)
```
Comment (FichteFoll) [inline overview/roles.md:93]: Don't these underscores cause text to be italicized?

**Review by FichteFoll — COMMENTED** [review]
Very good changes. Thanks for these.

Please just see a couple of my comments.

**Comment by FichteFoll** [comment]
>Perfect. Just make sure this page is accessible by at most two clicks from any page on the site.

It will in fact always be exactly one click.

File: typesetting/aegisub.md (hunk 3 above)
Comment (OrangeChannel) [reply typesetting/aegisub.md:35]: I was going to bother explaining it, updating some of the information [here](https://subarashii-no-fansub.github.io/Subbing-Tutorial/Tracking-Motion/), but until v2.80 comes out, I don't think it's worth using. If 2.80 makes it much easier to do (with an actual decent GUI this time) I could add it back. I recall the [export script](https://raw.githubusercontent.com/Subarashii-no-Fansub/AAE-Export/master/aae-export.py) not working with a beta version of the new Blender too, but I could be thinking of another script.

File: overview/roles.md (hunk 4 above)
Comment (OrangeChannel) [reply overview/roles.md:93]: Oddly enough, no they don't. It seems the caption plug-in doesn't recognize markdown formatting, and just italicizes everything. (The asterisks are just for an example, and aren't escaped)

image

Comment (OrangeChannel) [reply overview/roles.md:93]: This is on the descaling page for example, no formatting.

image

Comment (FichteFoll) [reply overview/roles.md:93]: Thanks for checking.

File: CONTRIBUTING.md (hunk 2 above)
Comment (FichteFoll) [reply CONTRIBUTING.md:350]: Addressed in 6c39f575d97f3431892d1b2d33b28d4a0d30edb4.

**Pull request merged**